### ISSUE TITLE
Icon comes before summary in daily/hourly blocks

### ISSUE THREAD

**Issue by cloneofghosts** [Pirate-Weather/pirateweather#378]
### Describe the bug

Something I noticed yesterday when testing the text summaries is that the text summary comes after the icon in the hourly and daily blocks. Everywhere else the summary comes before the icon and I'm not sure how long its been like that.

Just looked at an old archive of the Dark Sky docs and the summary is always before the icon.

### Expected behavior

Summary should always come before the icon

### Actual behavior

Summary comes after the icon in the daily/hourly blocks

### API Endpoint

Production

### Location

Ottawa, Ontario

### Troubleshooting steps

- [X] I have searched this repository and [Home Assistant Repository](https://github.com/alexander0042/pirate-weather-ha) to see if the issue has already been reported.
- [X] I have read through the [API documentation](https://pirateweather.net/en/latest/API/) before opening this issue.
- [X] I have not included my API key anywhere in my issue.
- [X] I have written an informative title.

**Comment by alexander0042** [comment]
Good catch here- the order technically shouldn't matter in JSON, but I think it's important to get right!

Fixed the order in 2.5c, so should be live on dev now!

**Comment by cloneofghosts** [comment]
I see that the order was fixed for hourly but daily was missed.

**Comment by alexander0042** [comment]
Ok, think I've fixed it now!

**Comment by cloneofghosts** [comment]
Yup, that fixed it.
